lib/apiservers/engine/backends/container.go

```diff
@@ -109,14 +109,14 @@ type Container struct {
 }
 
 const (
-	commitTimeout          = 1 * time.Minute
+	commitTimeout          = 3 * time.Minute
```

Comment:
I am in favor of getting rid of this timeout altogether. This is a client side timeout, and given that the caller of the docker command can hit Ctrl-C to back out of this, having an arbitrary timeout seems superfluous. The port layer server is going to (eventually) queue up an operation on VC, and that will result in a response in a finite amount of time. Any concerns about outstanding operations taking up resources indefinitely on the server side on the VCH should be mitigated by this.